osmosis-proto/Cargo.toml

```diff
@@ -1,17 +1,34 @@
 [package]
 name = "osmosis-proto"
-version = "0.0.0"
-authors = ["Tony Arcieri <tony@iqlusion.io>"]
+version = "1.0.0"
+authors = [
+    "Tony Arcieri <tony@iqlusion.io>",
+    "Philipp Schlütermann <philipp.schluetermann@udo.edu>"
+]
 license = "Apache-2.0"
 repository = "https://github.com/cosmos/cosmos-rust/tree/main/osmosis-proto"
-description = "Protobuf stuct defintions for interacting with Osmosis blockchains"
+description = "Protobuf stuct defintions for interacting with Osmosis"
 readme = "README.md"
 categories = ["cryptography", "cryptography::cryptocurrencies", "database"]
-keywords = ["blockchain", "cosmos", "tendermint", "proto"]
+keywords = ["blockchain", "cosmos", "osmosis", "tendermint", "proto"]
 edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
-cosmos-sdk-proto = { version = "0.12", path = "../cosmos-sdk-proto" }
```

Comment:
It would be good if the protos in the `cosmos` module could be sourced from the `cosmos-sdk-proto` crate, rather than duplicate copies of them being generated in the `osmosis-proto` crate.

The `cosmos-sdk-proto` crate does something similar with protos from the `tendermint-proto` crate.

Reply:
I fixed it with the commit [importing cosmos module cosmos-sdk-proto](https://github.com/cosmos/cosmos-rust/pull/239/commits/d53e64e68857e28fe15eb108fd0b7dc2d5e66ead)